packages/nitro-protocol/docs/interfaces/Adjudicator.md

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
Since the docs are generated from the code on `yarn build`, we can ignore `packages/nitro-protocol/docs` (or maybe `packages/nitro-protocol/docs/interfaces`?)